rhoas-cli-kafka/README.adoc

```diff
@@ -0,0 +1,283 @@
+----
+$ rhoas service-account create --name "my-service-acct" --file-format json
+
+Service account my-service-account created
+Credentials saved to /home/developer/my-project/credentials.json
+----
+
+Your service account credentials are created and saved to a JSON file.
+
+[NOTE]
+====
+When creating a service account, you can choose the file format and location to save the credentials.
+For more information, see the `rhoas service-account create` command help.
+====
+--
+
+. To verify your service account credentials,
+view the `credentials.json` file that you created.
++
+--
+.Verifying service account credentials
+[source,shell]
+----
+$ cat credentials.json
+{
+	"clientID":"srvc-acct-8c95ca5e1225-94a-41f1-ab97-aacf3df1",
+	"clientSecret":"facf3df1-ab97-2253-aa87-ab97",
+        "oauth_token_url": "https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token"
+}
+----
+
+You'll use these credentials and the bootstrap server URL to connect your applications and services to your Kafka instance.
+--
+
+[id="proc-creating-kafka-topic-cli_{context}"]
+== Creating a Kafka topic
+
+[role="_abstract"]
+After creating a Kafka instance, you can create Kafka topics to start producing and consuming messages in your services.
+
+.Prerequisites
+
+* You have created a Kafka instance, and it has a `ready` status.
+
+[NOTE]
+====
+You can use `rhoas` to switch to a specific Kafka instance.
+
+.Selecting a Kafka instance to use
+[source,shell]
+----
+$ rhoas kafka use --name my-kafka
+Kafka instance "my-kafka" has been set as the current instance.
+----
+====
+
+.Procedure
+
+. Create a Kafka topic with default values.
++
+--
+This example creates the `my-topic` Kafka topic.
+
+.Creating a Kafka topic with default values
+[source,shell]
+----
+$ rhoas kafka topic create --name my-topic
+{
+  "config": [
+    {
+      "key": "retention.ms",
+      "value": "-1"
+    }
+  ],
+  "name": "my-topic",
+  "partitions": [
+    {
+      "id": 1,
+      "leader": {
+        "id": 1
+      },
+      "replicas": [
+        {
+          "id": 1
+        }
+      ]
+    }
+  ]
+}
+----
+
+[NOTE]
+====
+If you do not want to use the default values,
+you can specify the number of partitions (`--partitions`) and message retention time (`--retention-ms`).
+For more information, use the command help `rhoas kafka topic create -h`.
+====
+--
+
+. If necessary, you can edit or delete the topic by using the `rhoas kafka topic update` and `rhoas kafka topic delete` commands.
+
+[id="proc-commands-managing-kafka_{context}"]
+== Commands for managing Kafka
+
+[role="_abstract"]
+For more information about the `rhoas` commands you can use to manage your Kafka instance,
+use the following command help:
+
+* `rhoas kafka -h` for Kafka instances
+* `rhoas service-account -h` for service accounts
+* `rhoas kafka topic -h` for Kafka topics
+
+[role="_additional-resources"]
+.Additional resources
+* {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^]
```

Comment:
I'm sure you're aware, but just a reminder that we'll need to define these attributes in the downstream docs repo so that the downstream version points to the CLI command ref on the customer portal.

Reply:
Thanks, Ben. Noted